src/anndata/_core/sparse_dataset.py

```diff
@@ -317,7 +317,10 @@ def get_memory_class(
     format: Literal["csr", "csc"], *, use_sparray_in_io: bool = False
 ) -> type[_cs_matrix]:
     if not CAN_USE_SPARSE_ARRAY and use_sparray_in_io:
-        msg = "scipy.sparse.cs{r,c}array is not available in current scipy version, falling back to scipy.sparse.spmatrix for reading"
+        msg = (
+            "scipy.sparse.cs{r,c}array is not available in current scipy version."
```

Comment:
don’t forget that you have to add a space after the first `.` in multiline messages